File: ITNews.Data.Contracts/Comments/Comment.cs
using ITNews.Data.Contracts.Users;
using System;
using System.Collections.Generic;
using System.Text;

namespace ITNews.Data.Contracts.Comments
{
    public class Comment : EntityBase
    {
        public string Content { get; set; }
        public virtual NewsComment NewsComment { get; set; }
        public DateTime CreatedDate { get; set; }
        public virtual User Author { get; set; }
        public int AuthorId { get; set; }
    }
}

File: ITNews/wwwroot/js/Comment/Comment.js
let Comment = function (id, content, createdDate, newsId, authorId, author) {
    let self = this;
    self.id = id;
    self.content = content;
    self.createdDate = new Date(createdDate).toLocaleString();
    self.newsId = newsId;
    self.authorId = authorId;
    self.author = author
        ? new User(author.Id, author.Email, undefined, author.Name, author.Photo)
        : undefined;
}